src/foam/nanos/auth/Group.js

```diff
@@ -38,5 +44,25 @@ foam.CLASS({
       documentation: 'Custom authentication settings for this group.'
     }
     */
+  ],
+  methods: [
+    {
+      name: 'implies',
+      javaReturns: 'Boolean',
+      args: [
+        {
+          name: 'permission',
+          javaType: 'java.security.Permission'
+        }
+      ],
+      javaCode:
```

Comment:
We should add this to the style guide as well. Seems throughout the project all javaCode has concatenation